NumLib/ODESolver/TimeDiscretization.h

```diff
@@ -220,12 +220,18 @@ class BackwardEuler final : public TimeDiscretization
     {
         _t = t0;
         MathLib::LinAlg::copy(x0, _x_old);
+        // The function only has computation if DDC is appied,
+        // e.g. Parallel comuting.
+        MathLib::LinAlg::setLocalAccessibleVector(_x_old);
```

Comment:
I think this is not needed in BackwardEuler, because _x_old is only used in the MatrixTranslator class to compute some matrix-vector-products. (As far as I can see.)